extensions/wikia/AttributeServiceMWApi/AttributeServiceMWApi.setup.php

```diff
@@ -1,9 +0,0 @@
-<?php
```

Comment:
Reverted this AttributeServiceMWApiController nonsense since we're taking a different approach.